### DISCUSSION TITLE
Extension / Apps unable to view organizational ciphers / collections.

### DISCUSSION THREAD

**Discussion by ruifung** [dani-garcia/vaultwarden#2620] [Q&A]
### Subject of the issue

When viewing from the mobile app / web extension / windows app,
any entries belonging to a shared organization doesn't show up.

They appear just fine in the Web Vault though.

### Deployment environment

It should be noted that the below diagnostics data has some inaccuracies.
1. It is running in a container, though in a kubernetes cluster.

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.25.0
* Web-vault version: v2.28.1
* Running within Docker: false (Base: Debian)
* Environment settings overridden: true
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: PostgreSQL
* Database version: PostgreSQL 14.4 on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
* Clients used: Browser Extensions (Chrome/Firefox), Windows app, Web Vault
* Reverse proxy and version: CloudFlare / Traefik
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>

**Environment settings which are overridden:** DOMAIN, SIGNUPS_ALLOWED, ADMIN_TOKEN, SMTP_HOST, SMTP_PORT, SMTP_FROM, SMTP_USERNAME, SMTP_PASSWORD

```json
{
  "_duo_akey": "***",
  "_enable_duo": true,
  "_enable_email_2fa": true,
  "_enable_smtp": true,
  "_enable_yubico": true,
  "_ip_header_enabled": true,
  "admin_ratelimit_max_burst": 3,
  "admin_ratelimit_seconds": 300,
  "admin_token": "***",
  "allowed_iframe_ancestors": "",
  "attachments_folder": "/bucket/attachments",
  "authenticator_disable_time_drift": false,
  "data_folder": "/data",
  "database_conn_init": "",
  "database_max_conns": 10,
  "database_timeout": 30,
  "database_url": "**********://***********:************^^***$*******@**********/***********",
  "db_connection_retries": 15,
  "disable_2fa_remember": false,
  "disable_admin_token": false,
  "disable_icon_download": false,
  "domain": "*****://*********.***.**",
  "domain_origin": "*****://*********.***.**",
  "domain_path": "",
  "domain_set": true,
  "duo_host": "***.duosecurity.com",
  "duo_ikey": "***",
  "duo_skey": "***",
  "email_attempts_limit": 3,
  "email_expiration_time": 600,
  "email_token_size": 6,
  "emergency_access_allowed": true,
  "emergency_notification_reminder_schedule": "0 5 * * * *",
  "emergency_request_timeout_schedule": "0 5 * * * *",
  "enable_db_wal": true,
  "extended_logging": true,
  "helo_name": null,
  "hibp_api_key": "***",
  "icon_blacklist_non_global_ips": true,
  "icon_blacklist_regex": null,
  "icon_cache_folder": "/tmp/iconcache",
  "icon_cache_negttl": 259200,
  "icon_cache_ttl": 2592000,
  "icon_download_timeout": 10,
  "icon_redirect_code": 302,
  "icon_service": "internal",
  "incomplete_2fa_schedule": "30 * * * * *",
  "incomplete_2fa_time_limit": 3,
  "invitation_org_name": "RF BitWarden",
  "invitations_allowed": true,
  "ip_header": "X-Real-IP",
  "job_poll_interval_ms": 30000,
  "log_file": null,
  "log_level": "warn",
  "log_timestamp_format": "%Y-%m-%d %H:%M:%S.%3f",
  "login_ratelimit_max_burst": 10,
  "login_ratelimit_seconds": 60,
  "org_attachment_limit": null,
  "org_creation_users": "",
  "password_iterations": 100000,
  "reload_templates": false,
  "require_device_email": true,
  "rsa_key_filename": "/data/rsa_key",
  "send_purge_schedule": "0 5 * * * *",
  "sends_allowed": true,
  "sends_folder": "/bucket/sends",
  "show_password_hint": true,
  "signups_allowed": false,
  "signups_domains_whitelist": "",
  "signups_verify": false,
  "signups_verify_resend_limit": 6,
  "signups_verify_resend_time": 3600,
  "smtp_accept_invalid_certs": false,
  "smtp_accept_invalid_hostnames": false,
  "smtp_auth_mechanism": null,
  "smtp_debug": false,
  "smtp_explicit_tls": false,
  "smtp_from": "*********@**.***.**",
  "smtp_from_name": "RF BitWarden",
  "smtp_host": "****.*******.***",
  "smtp_password": "***",
  "smtp_port": 587,
  "smtp_security": "starttls",
  "smtp_ssl": true,
  "smtp_timeout": 15,
  "smtp_username": "*********@**.***.**",
  "templates_folder": "/data/templates",
  "tmp_folder": "/bucket/uploads",
  "trash_auto_delete_days": null,
  "trash_purge_schedule": "0 5 0 * * *",
  "use_syslog": false,
  "user_attachment_limit": null,
  "web_vault_enabled": true,
  "web_vault_folder": "web-vault/",
  "websocket_address": "0.0.0.0",
  "websocket_enabled": true,
  "websocket_port": 3012,
  "yubico_client_id": null,
  "yubico_secret_key": null,
  "yubico_server": null
}
```
</details>

### Steps to reproduce

1. Log in on a device.
3. Attempt to access organizational ciphers.

### Expected behaviour

The entries to appear.

### Actual behaviour

The app/extension shows that there are no entries.

### Troubleshooting data

I've recently migrated from mariadb to postgresql, but I don't think that should affect it right? Given how the data is showing up in the web vault.
![image](https://user-images.githubusercontent.com/1073945/178925937-e1f0a167-e994-4b9b-b0a7-d56038cb1914.png)

**Comment by BlackDex** [comment]
It works fine for me.

And if they show in the web-vault, then it should also be in all other apps, since they all use the same API calls, and all receive the same data.

I would suggest to fully logout and back in again.

**Comment by ruifung** [comment]
I'm not sure, but I must have accidentally fouled up something while migrating databases somehow, that apparently only affected everything except the web vault.

Because after doing the following procedure:
1. Assign a new organization owner
2. Leave the organization
3. Use the new organization owner to re-invite me back

The entries now show up normally. Which is pretty weird that it only affected the apps and not the web vualt.
